spells/swap-team-rack.js

```diff
@@ -0,0 +1,40 @@
+function swapTeamRack(payload){
+
+    // Unpack the values
+    [station, team] = payload;
+
+    // Get the player's id
+    let playerId = context.id;
+
+    // Get the player's state so that we can get their team
+    let player = context.getPlayerState(playerId);
+
+    // Get the player's team
+    let playerTeam = player.team;
```

Comment:
Getting the player team should most likely be done by:

``` js
const playerId = context.id();
const game = context.getGameState();
const player = game.players[playerId];
const teamIndex = player.team;
```

Reply:
Since the team index is stored within the player (inside of a game) not inside the player state (which is the global state on the server **not** coupled with a game).